tests/usbus/tests/01-run.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python3
+
+# Copyright (C) 2016 Kaspar Schleiser <kaspar@schleiser.de>
+# Copyright (C) 2016 Takuo Yonezawa <Yonezawa-T2@mail.dnp.co.jp>
```

Comment:
Hmmmmm, really ?

Reply:
Yes, really, I copied this file from one of the other tests. I didn't change the file significantly so I don't think I should claim copyright.